workers/data_refinery_workers/processors/smashing_utils.py

```diff
@@ -369,8 +373,16 @@ def process_frames_for_key(key: str,
                 elif frame['technology'] == 'rnaseq':
                     rnaseq_columns.extend(frame['dataframe'].columns)
 
-        all_gene_identifiers = list(all_gene_identifiers)
+        total_samples = len(microarray_columns) + len(rnaseq_columns)
+        all_gene_identifiers = []
+        for gene_id, sample_count in gene_identifier_counts.items():
+            # We only want to use gene identifiers which are present
+            # in >50% of the samples.
+            if sample_count > total_samples / 2:
```

Comment:
Do you want to convert this to the same methodology as the other filtering steps (`* 0.5`) instead of divide by 2? Should be the same, but I think in the compendia we always multiply by the decimal fraction.